src/main/java/io/aiven/kafka/connect/s3/S3SinkConfig.java

```diff
@@ -88,39 +88,22 @@
     public static final String TIMESTAMP_TIMEZONE = "timestamp.timezone";
     public static final String TIMESTAMP_SOURCE = "timestamp.source";
 
-    public static final Set<String> OUTPUT_FILED_NAMES = new HashSet<>() {
-        {
-            add(OUTPUT_FIELD_NAME_KEY);
-            add(OUTPUT_FIELD_NAME_OFFSET);
-            add(OUTPUT_FIELD_NAME_TIMESTAMP);
-            add(OUTPUT_FIELD_NAME_VALUE);
-            add(OUTPUT_FIELD_NAME_HEADERS);
-        }
-    };
     public static final String AWS_ACCESS_KEY_ID_CONFIG = "aws.access.key.id";
     public static final String AWS_SECRET_ACCESS_KEY_CONFIG = "aws.secret.access.key";
     public static final String AWS_S3_BUCKET_NAME_CONFIG = "aws.s3.bucket.name";
     public static final String AWS_S3_ENDPOINT_CONFIG = "aws.s3.endpoint";
     public static final String AWS_S3_PREFIX_CONFIG = "aws.s3.prefix";
     public static final String AWS_S3_REGION_CONFIG = "aws.s3.region";
-    public static final String FILE_NAME_PREFIX_CONFIG = "file.name.prefix";
-    public static final String FILE_NAME_TEMPLATE_CONFIG = "file.name.template";
-    public static final String FILE_COMPRESSION_TYPE_CONFIG = "file.compression.type";
-    public static final String FILE_MAX_RECORDS = "file.max.records";
-    public static final String FILE_NAME_TIMESTAMP_TIMEZONE = "file.name.timestamp.timezone";
-    public static final String FILE_NAME_TIMESTAMP_SOURCE = "file.name.timestamp.source";
-    public static final String FORMAT_OUTPUT_FIELDS_CONFIG = "format.output.fields";
-    public static final String FORMAT_OUTPUT_FIELDS_VALUE_ENCODING_CONFIG = "format.output.fields.value.encoding";
-    public static final String NAME_CONFIG = "name";
+
+    private static final String GROUP_AWS = "AWS";
+    private static final String GROUP_FILE = "File";
+    private static final String GROUP_FORMAT = "Format";
+
     private static final Logger LOGGER = LoggerFactory.getLogger(S3SinkConfig.class);
     // FIXME since we support so far both old style and new style of property names
     //      Importance was set to medium,
     //      as soon we will migrate to new values it must be set to HIGH
     //      same for default value
```

Comment:
Could you please move this comment together with the lines it was attached to?

Reply:
Right!
- [ ] Move comment to the top